### ISSUE TITLE
Remove the link for downloading photos from the non-staff view

### ISSUE THREAD

**Issue by juliandsmith** [AASHE/hub#311]
To discourage unauthorized use of submitted photographs, the "Download" link on photograph pages should be removed for all but logged in staff users.

**Comment by monika-aashe** [comment]
Agreed to include in other image feature milestone along with related improvements.

**Comment by monika-aashe** [comment]
Will revisit after the SCI photo request to see if this needs to happenl

**Comment by jamstooks** [comment]
The reason we bike racked this, is because there are images, like [this infographic](https://hub.aashe.org/browse/outreachmaterial/14595/STARS-Silver-for-NAIT), that are better viewed at full-size. It makes sense to provide the full-size image here and probably in many other cases.

Perhaps we could add some sort of not about valid use?

**Comment by monika-aashe** [comment]
There is a workaround to the download tab - right click - open in new window. I don't recall having issues with this in the past so okay as is for now.
